crates/python/src/compiler/quilc.rs

```diff
@@ -136,8 +136,10 @@ impl PyQuilcClient {
 }
 
 py_function_sync_async! {
+    #[pyo3_opentelemetry::pypropagate(on_context_extraction_failure = "print")]
```

Comment:
Why do we print the failures on these instead of the default?

Reply:
I actually changed this upstream to make "print" the default behavior, so once I update the version here, I can drop these.